src/main/groovy/net/wasdev/wlp/gradle/plugins/tasks/StartTask.groovy

```diff
@@ -32,6 +32,8 @@ class StartTask extends AbstractTask {
         }
         executeServerCommand(project, 'start', params)
         
+        copyConfigFiles()
```

Comment:
This should be handled by dependsOn libertyCreate.  I updated task https://github.ibm.com/was-liberty/ecosystem/issues/116 to handle this change.  Remove this call to copyConfigFiles, so that we don't end up with it in here twice.